commit 86c05835257c9e211c54f13cc7fd9a8669a790f9 (from 6cce924cd0c1de1125a893c83555e9abb5aecc16)
Merge: 6cce924 0c24bb6
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Tue Feb 3 08:29:03 2015 -0800

    Merge pull request #1459 from AnHardt/lcd-width
    
    Adjust LCD_WIDTH to make better use of DOGM LCDs 2

diff --git a/Marlin/Configuration.h b/Marlin/Configuration.h
index ceaa72a..6f8d238 100644
--- a/Marlin/Configuration.h
+++ b/Marlin/Configuration.h
@@ -709,7 +709,7 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
   #define SDSUPPORT
   #define ULTRA_LCD
   #ifdef DOGLCD // Change number of lines to match the DOG graphic display
-    #define LCD_WIDTH 20
+    #define LCD_WIDTH 22
     #define LCD_HEIGHT 5
   #else
     #define LCD_WIDTH 20
@@ -718,7 +718,7 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
 #else //no panel but just LCD
   #ifdef ULTRA_LCD
   #ifdef DOGLCD // Change number of lines to match the 128x64 graphics display
-    #define LCD_WIDTH 20
+    #define LCD_WIDTH 22
     #define LCD_HEIGHT 5
   #else
     #define LCD_WIDTH 16

